ListUtil.xs

```diff
@@ -1204,6 +1204,52 @@ CODE:
     XSRETURN(items);
 }
 
+void
+sample(...)
+PROTOTYPE: @
+CODE:
+{
+    UV count = items ? SvUV(ST(0)) : 0;
+    int reti = 0;
+    SV *randsv = get_sv("List::Util::RAND", 0);
+    CV * const randcv = randsv && SvROK(randsv) && SvTYPE(SvRV(randsv)) == SVt_PVCV ?
```

Comment:
For performance if count is 0 (silly) might be better to grab randcv below `if (!count) XSRETURN(0)` but seems rare enough to probably not be worth it.

Reply:
That's possible but then (because C89) the variable needs declaring on a separate line to assignment. Not worth the readability hit for that tiny performance improvement in a rare case.